src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -179,7 +181,7 @@ private function createPasswordQuestion()
 
         return $passwordQuestion->setValidator(function ($value) {
             if ('' === trim($value)) {
-                throw new \Exception('The password must not be empty.');
+                throw new InvalidArgumentException('The password must not be empty.');
```

Comment:
@chalasr It feels like a visual bug, so we're fixing the visual representation of the message, what do you think?